webapp/locales/en.json

```diff
@@ -159,6 +159,27 @@
       "labelBio": "About You",
       "success": "Your data was successfully updated!"
     },
+    "email": {
+      "validation": {
+        "same-email": "This is your current email address"
+      },
+      "name": "Your email",
+      "labelEmail": "Change your email address",
+      "labelNewEmail": "New email Address",
+      "labelNonce": "Enter your code",
+      "success": "A new email address has been registered.",
+      "submitted": "An email to verify your address has been sent to <b>{email}</b>.",
+      "change-successful": "Your email address has been changed successfully.",
+      "verification-error": {
+        "message": "Your email could not be changed.",
+        "explanation": "This can have different causes:",
+        "reason": {
+          "invalid-nonce": "Is the confirmation code invalid?",
+          "no-email-request": "Are you certain that you requested a change of your email address?"
+        },
+        "support": "If the problem persists, please contact us by email at"
+      }
```

Comment:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2019-09-30T09:14:54Z" title="Monday, September 30th 2019, 11:14:54 am +02:00">Sep 30, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -158,6 +158,28 @@
       "labelBio": "About You",
       "success": "Your data was successfully updated!"
     },
+    "email": {
+      "validation": {
+        "same-email": "Must be different from your current E-Mail address"
+      },
+      "name": "Your E-Mail",
+      "labelEmail": "Change your E-Mail address",
+      "labelNewEmail": "New E-Mail Address",
+      "labelNonce": "Enter your code",
+      "success": "A new E-Mail address has been registered.",
+      "submitted": "An email to verify your address has been sent to <b>{email}</b>.",
+      "change-successful": "Your E-Mail address has been changed successfully.",
+      "verification-error": {
+        "message": "Your E-Mail could not be changed.",
+        "explanation": "This can have different causes:",
+        "reason": {
+          "invalid-nonce": "Is the confirmation code invalid?",
+          "no-email-request": "You haven't requested a change of your email address at all?",
+          "email-address-taken": "Has the email been assigned to another user in the meantime?"
+        },
+        "support": "If the problem persists, please contact us by e-mail at"
```


I think last time we decided to go with the spelling `e-mail` in English, or am I mistaken @Tirokk? Would be great to have it all the same – and there are some `E-Mail`s and `email`s in this file. 😜

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-10-01T16:50:14Z" title="Tuesday, October 1st 2019, 6:50:14 pm +02:00">Oct 1, 2019</time>_

---


I think we didn't decide... @Tirokk asked it to be changed in your `PR` cause it's what _he_ prefers, and since no one else cared enough to point it out, it's what we went with. I prefer the non-hyphenated version personally. While `e-mail` is still more commonly used than `email`, it's on the downturn, so if we wanna keep up with the times, maybe we should be cool and use `email` :stuck_out_tongue_closed_eyes: 
https://writingexplained.org/e-mail-or-email

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-10-01T22:53:02Z" title="Wednesday, October 2nd 2019, 12:53:02 am +02:00">Oct 2, 2019</time>_

---


So how should I change it? `email` now?

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-10-01T22:54:45Z" title="Wednesday, October 2nd 2019, 12:54:45 am +02:00">Oct 2, 2019</time>_

---


In German it seems to `E-Mail` and nothing else: https://www.duden.de/sprachwissen/sprachratgeber/Schreibung-von-E-Mail

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-10-02T07:51:50Z" title="Wednesday, October 2nd 2019, 9:51:50 am +02:00">Oct 2, 2019</time>_

---


I say just leave it, like I said, it's not something I would ask to be changed. 
Sorry, I just couldn't help myself when @alina-beck said we decided... `e-mail` is perfectly ok, there seem to be stricter rules in German than English and as long as it's not incorrect, we can follow the stricter rules to have some continuity between the two.

Reply:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2019-10-02T09:22:16Z" title="Wednesday, October 2nd 2019, 11:22:16 am +02:00">Oct 2, 2019</time>_

---


I really don't mind if we go with `email` or `e-mail` in English! (Personally I've only been using `email` so far...) But I think it would be great if we could agree on one and then stick to it in all user-facing communication.

It's not a big issue and most people won't notice, I guess, but some (language :nerd_face: like me) might and it's just overall smoother and cleaner. :wink:

Since I don't have a strong opinion I just went with what @Tirokk suggested last time. But what do you say @mattwr18? Would `email` be better? Let's decide! :smile:

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-10-02T14:35:00Z" title="Wednesday, October 2nd 2019, 4:35:00 pm +02:00">Oct 2, 2019</time>_

---


I wouldn't say it would be better, I don't think it really matters one way or the other, and I completely understand for user facing communication it's great to be consistent. It's a little funny to me cause all of our `keys` in the `locale` files we spell it `email` not `e-mail`, but those are not user facing.
I'm happy to use `e-mail` for the reasons I stated above :point_up: 
Now, we can probably freely say we decided :rofl: